pkg/controller/pcidevice/pcidevice_controller.go

```diff
@@ -61,17 +63,26 @@ func Register(
 
 func (h Handler) reconcilePCIDevices(nodename string) error {
 	// List all PCI Devices on host
+	pci, err := ghw.PCI()
```

Comment:
lookup of PCI devices is not needed, as the `Register` method is already handling the same

Reply:
change it to the `h.pci` right?

Reply:
👍

Reply:
actually this is not needed at all. since we already are reconcilling and storing state of pci devices in the handler. So we need to range over h.pci.